Sources/MySQLKit/MySQLDatabase.swift

```diff
@@ -201,4 +203,9 @@ public struct MySQLDialect: SQLDialect {
             alterColumnDefinitionTypeKeyword: nil
         )
     }
+    
+    public func normalizeSQLConstraint(identifier: SQLExpression) -> SQLExpression {
+        guard let sqlIdentifier = identifier as? SQLIdentifier else { return identifier }
+        return SQLRaw(String(describing: Insecure.SHA1.hash(data: Data(sqlIdentifier.string.utf8))))
```

Comment:
Since SHA-1 is _known_ to result in conflicts, it seems that using it to create constraints could be an issue?